lib/tasks/user.rake

```diff
@@ -10,4 +10,11 @@ namespace :user do
     end
     puts 'The accounts\' first_admin_id have been updated.'
   end
+
+  task :obfuscate_private_and_repair_tenant, [:user_ids] => [:environment] do |_task, args|
+    User.where(id: args[:user_ids]).find_each do |user|
+      user.update!({username: "someone#{user.id}", email: "someone#{user.id}@example.com", tenant_id: user.account.provider_account.id}, without_protection: true)
```

Comment:
`provider_account.id` will perform an extra SELECT to the db. Was this intentional, perhaps to confirm the provider account exists? If we can trust the user's account always has a valid provider, we can use `user.account.provider_account_id` instead and save one request to the db.